src/components/Navigation/LanguageDropdown.js

```diff
@@ -31,15 +31,16 @@ const LanguageDropdown = ({mode, language, resetFilter, setLanguage, t}) => {
                     noCaret
                     pullRight
                     bsStyle={'language-' + mode}
-                    title={<LanguageItem language={language} t={t}/>}
+                    title={<LanguageItem style={[styles.language, styles.onlyFlag].join(' ')}
```

Comment:
Bedre, men jeg synes kanskje at det er LanguageItem som selv skal velge hvilke styles som skal brukes. Føler at litt for mye av detaljene blir liggende utenfor LanguageItem dersom det er LanguageDropdown-komponenten som skal velge styles-keys for LanguageItem.

Mulig "onlyFlag" som boolean ikke er beste løsning, men synes det ville vært bedre... men om du mener måten du har gjort det på er bedre, må du gjerne argumentere for det! ;)